python/tests/__init__.py

```diff
@@ -0,0 +1,7 @@
+import unittest
+
+
+def test_suite():
+    loader = unittest.TestLoader()
+    suite = loader.discover('python/tests', pattern='test*.py')
```

Comment:
FYI, this hard-coded path meant the tests can only be run from the root of the repository: I fixed it with:
https://github.com/BlueBrain/libsonata/pull/106